README.md

```diff
@@ -7,20 +7,20 @@ This repo contains the sources of the app, netlify functions and configuration f
 ### 1 Requirements:
 
 * Node.js version 10.x: [latest](https://nodejs.org/en/download/current/)
-* Angular CLI: `npm install -g @angular/cli`
-* Netlify lambda CLI : `npm install netlify-lambda`
+* Angular CLI: `yarn global add @angular/cli`
+* Netlify lambda CLI : `yarn add netlify-lambda`
```

Comment:
I think you can remove this line, because `netlify-lambda` is already part of the dev dependencies, right?